### PR TITLE
Feat/log/uuid

### PR DESCRIPTION
**Pull request by ForrestWeston**
Add an uuid/requestId to root context - this will add a requestId to event logs resulting from a command
Add event logs around path resolution - said logs make use of requestId

### CODE REVIEW

File: path/resolver.go [hunk 1]
```diff
@@ -135,6 +135,7 @@ func (s *Resolver) ResolvePathComponents(ctx context.Context, fpath Path) ([]nod
 	if err != nil {
 		return nil, err
 	}
+	defer log.EventBegin(ctx, "resolvePathComponents", logging.LoggableMap{"parts": parts, "cid": h}).Done()
```
Comment (ForrestWeston) [inline path/resolver.go]: In the case that there is an error in the execution of this method, do we want to add a loggable error to the event log (see L143)?

Comment (Stebalien) [reply path/resolver.go]: IMO, yes. We'd probably also want to log the error from L146 as well. Also, this event should probably go at the top and include any errors from `SplitAbsPath`.

Comment (Kubuxu) [reply path/resolver.go]: You can save the result value of `EventBegin` and add additional data to it.

File: commands/http/handler.go [hunk 2]
```diff
@@ -137,6 +138,7 @@ func (i internalHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	}
 
 	ctx, cancel := context.WithCancel(node.Context())
+	ctx = logging.ContextWithLoggable(ctx, loggables.Uuid("requestId"))
```
Comment (whyrusleeping) [inline commands/http/handler.go]: I would move this down below the defer cancel.

**Comment by whyrusleeping** [comment]
@VictorBjelkholm whatsup with jenkins?

**Pull request merged**